control-plane/csi-driver/src/context.rs

```diff
@@ -104,15 +85,23 @@ impl Parameters {
     pub fn sts_affinity_group(value: Option<&String>) -> Result<Option<bool>, ParseBoolError> {
         Self::parse_bool(value)
     }
+    /// Parse the value for `Self::CloneFsAsIdVolumeId`
+    pub fn clone_fs_id_as_volume_id(
+        value: Option<&String>,
+    ) -> Result<Option<bool>, ParseBoolError> {
+        Self::parse_bool(value)
+    }
 }
 
 /// Volume publish parameters.
 #[allow(dead_code)]
+#[derive(Debug)]
 pub struct PublishParams {
     io_timeout: Option<u32>,
     ctrl_loss_tmo: Option<u32>,
     keep_alive_tmo: Option<u32>,
     fs_type: Option<FileSystem>,
+    change_fs_id: Option<bool>,
```

Comment:
Why not just do this, and let the csi-controller do the checks for clone/restore?
Then csi-node just knows if it has to ensure fs needs to have this id or not, and doesn't need to know about clones/restores.

```suggestion
    fs_id: Option<String>,
```

Reply:
This sounds good. Thanks.